includes/machine-learning-service-inference-config.md

```diff
@@ -34,4 +35,32 @@ The following JSON is an example inference configuration for use with the CLI:
     "baseImage": null,
     "baseImageRegistry": null
 }
-```
\ No newline at end of file
+```
+
+The following JSON is an example inference configuration that uses an existing Azure Machine Learning [environment](https://docs.microsoft.com/python/api/azureml-core/azureml.core.environment.environment?view=azure-ml-py) with a specific version for use with the CLI:
+
+```json
+{
+    "entryScript": "score.py",
+    "environment":{
+        "name": "myenv",
+        "version": "1"
+    },
+    "condaFile": "myenv.yml",
```

Comment:
this will hit the validation error, i believe. https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=%2Fsrc%2Fazureml-core%2Fazureml%2Fcore%2Fmodel.py&version=GBmaster&line=1975&lineEnd=1979&lineStartColumn=1&lineEndColumn=119&lineStyle=plain